src/sidebar/BrowseView.js

```diff
@@ -10,15 +10,6 @@ import { decode } from "../util.js";
 
 import Template from "./BrowseView.blp" with { type: "uri" };
 
-const IGNORED_LIBRARIES = [
-  "atk",
-  "javascriptcoregtk-4.1",
-  "libhandy-1",
-  "libnotify-0",
-  "webkit2gtk-4.1",
-  "webkit2gtk-web-extension-4.1",
```

Comment:
These are outdated docs though and might be misleading/conflicting, we could keep ``atk`` but gtk already supports accessibility stuff well

Reply:
isn't webkit4.1 the current version of webkitgtk for GTK 3?

Reply:
Ditto atk and libhandy

Reply:
But I think right now we only want to support GTK 4 stuff? We do have this issue open though https://github.com/workbenchdev/Biblioteca/issues/22. I dont mind including them but there's no way to handle conflicting entries while searching, so maybe we can include them once we have support to display doc versions in the search results.

Reply:
Might make sense to not show the old webkitgtk docs then Regarding the others up to you, but it probably does not hurt to have them as long as they use gidocgen and do not have duplicated api.

Reply:
Hmm okay I see, I thought libhandy, libnotify and atk weren't used anymore but ig we can include those

Reply:
atk and handy are GTK 3 only. Libnotify on the other hand is not something GTK apps should use since GNotification covers the same usecase and is part of GLib.

Reply:
Just for the sake of being precise, none of the libraries in that list are deprecated, with the exception of maybe javascriptcoregtk-4.1.

Reply:
I'm totally fine with including everything with the exception of the webkit 4.2 for the reasons I mentioned above about searching and duplicated APIs. The user defined filter idea is interesting and might be something we want, but I think this should do for now.

Reply:
If there are no docs for GTK 3 itself, then yeah makes sense to filter all GTK 3 specific docs.

Reply:
Okay, so we'll keep everything as is and filter everything like before then?